frontend/javascripts/oxalis/model/bucket_data_handling/bucket_picker_strategies/flight_bucket_picker.js

```diff
@@ -140,11 +142,14 @@ export default function determineBucketsForFlight(
   };
 
   const fallbackBuckets = isFallbackAvailable ? traverseFallbackBBox(getBBox(planeBuckets)) : [];
-  traversedBuckets = traversedBuckets.concat(fallbackBuckets);
+  // Use a constant priority for the fallback buckets which is higher than the highest non-fallback priority
+  const fallbackPriority = iterStep + 1;
```

Comment:
Maybe I'm misunderstanding the code, but isn't `halfWidth + halfWidth + xOffset` the highest non-fallback priority (line 86)?